### PR TITLE
Remove linking with boost_iostreams

### PR DESCRIPTION
**Pull request by stsoe**

### CODE REVIEW

**Comment by stsoe** [comment]
retest this please

**Comment by abalasa** [comment]
retest this please

**Comment by abalasa** [comment]
retest this please

File: src/xma/src/xmaapi/CMakeLists.txt [hunk 1]
```diff
@@ -38,7 +38,6 @@ target_link_libraries(xma2api
   stdc++
   xml2
   boost_filesystem
-  boost_iostreams
```
Comment (sarab96) [inline src/xma/src/xmaapi/CMakeLists.txt]: I'm look at this requirement. If I can get rid of this.

Comment (stsoe) [reply src/xma/src/xmaapi/CMakeLists.txt]: Please let me know.  I am baffled why this dependency was added, there is nothing in the code that depends on it.

Comment (sarab96) [reply src/xma/src/xmaapi/CMakeLists.txt]: Ok please merge this change. I made small change and this is no longer required.

**Pull request merged**